index.html

```diff
@@ -43,35 +43,28 @@
 
       <div class="row justify-content-center">
         <div class="col-xl-9 d-flex align-items-center justify-content-between">
-          <h1 class="logo"><a href="index.html">KnightOne</a></h1>
+          <h1 class="logo"><a href="index.html">ProffNITT</a></h1>
           <!-- Uncomment below if you prefer to use an image logo -->
           <!-- <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->
 
           <nav class="nav-menu d-none d-lg-block">
             <ul>
               <li class="active"><a href="index.html">Home</a></li>
               <li><a href="#about">About</a></li>
-              <li><a href="#services">Services</a></li>
-              <li><a href="#portfolio">Portfolio</a></li>
-              <li><a href="#pricing">Pricing</a></li>
-              <li class="drop-down"><a href="">Drop Down</a>
+              <li><a href="#members">Members</a></li>
+              <li><a href="#bharatx">BharatX</a></li>

```

Comment:
Don't add bharatx in the navbar itself, either put it under some "projects" tab or don't add it at all.